test/e2e/basic_workflow_test.go

```diff
@@ -132,6 +153,14 @@ func (test *e2eTest) serviceUpdate(t *testing.T, serviceName string, args []stri
 	assert.Check(t, util.ContainsAll(out, expectedOutput))
 }
 
+func (test *e2eTest) serviceDescribeWithPrintFlags(t *testing.T, serviceName string) {
+	out, err := test.kn.RunWithOpts([]string{"service", "describe", serviceName, "-o=name"}, runOpts{})
+	assert.NilError(t, err)
+
+	expectedName := fmt.Sprintf("service.serving.knative.dev/%s", serviceName)
+	assert.Check(t, strings.Contains(out, expectedName))
```

Comment:
this should be an exact string match IMO, i..e `assert.Equal(t, out, expectedName)`

Reply:
as this is not string manipulation and printing on the client side, but machine readable output